src/Components/Onboarding/Steps/Artists/ItemLink.tsx

```diff
@@ -38,28 +39,71 @@ const Col = styled.div`
   align-items: center;
 `
 
-interface State {
+export interface State {
   fadeIn: boolean
   fadeOut: boolean
+  artist_id: string
 }
 
-type Props = React.HTMLProps<HTMLAnchorElement> & RelayProps
+interface Props extends React.HTMLProps<HTMLAnchorElement>, RelayProps {
+  relay?: RelayProp
+}
 
 class ItemLink extends React.Component<Props, State> {
   state = {
     fadeIn: false,
     fadeOut: false,
+    artist_id: this.props.artist.id,
+  }
+
+  followArtist() {
```

Comment:
@xtina-starr @mzikherman Here’s the initial setup for the follow artist mutation.